cypress/integration/rendering/sequencediagram.spec.js

```diff
@@ -79,6 +79,15 @@ describe('Sequence diagram', () => {
       `
     );
   });
+  it('should render links', () => {
+    imgSnapshotTest(
+      `
+      sequenceDiagram
+      Alice->John: Look at [MermaidJS](https://mermaid.js.org/)
+      John->>Alice: Thanks for the link
+      `
+    );
+  });
   it('should handle different line breaks', () => {
```

Comment:
We need more test cases, with the edge cases that have been discussed in previous comments.